tests/compile-only/fail-broken-if-conditional.txt

```diff
@@ -0,0 +1 @@
+(6) : error 054: unmatched closing brace ("}")
```

Comment:
this isn’t error `142`, is this intended?

Reply:
To avoid error spam, the compiler only reports the first error on a line, and then it silences errors up until the next statement. So error 142 gets reported, but another error gets there first. (Arguably I should have re-used 29 since we're running out of error codes, but whatever.)

This is not ideal from a testing perspective but the parser is so incredibly brittle that not crashing is probably the best we can hope for.